From: Nikolay Martynov <(none)>
Date: Sun, 18 Jun 2017 21:14:57 -0400
Subject: Fix Makefile.am to get nodejs stuff installed

---
 auth-dialog/Makefile.am | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/auth-dialog/Makefile.am b/auth-dialog/Makefile.am
index b9fc086..f4be609 100644
--- a/auth-dialog/Makefile.am
+++ b/auth-dialog/Makefile.am
@@ -28,3 +28,13 @@ nm_openconnect_auth_dialog_LDADD = \
 
 CLEANFILES = *~
 
+EXTRA_DIST = auth-firefox
+
+dist-hook:
+	rm -rf `find $(distdir)/auth-firefox -type d -name '~*'`
+
+install-data-local:
+	cp -Rv auth-firefox $(DESTDIR)$(libexecdir)
+
+uninstall-hook:
+	rmdir $(libexecdir)/auth-firefox
